### ISSUE TITLE
robot model change Day #19

### ISSUE THREAD

**Issue by WROMagnet** [WROMagnet/Magnet-Robots#45]
Today, April 15, 2024, we are at @LuisriosWRO3's grandmother's house. We will be changing the model of the robot since we have to assemble the turning mechanism and we plan to remove a floor from the robot.

- Today's goal is to completely disassemble the robot and make a single-story robot with the front turning mechanism.

**Comment by WROMagnet** [comment]
- We have already dismantled the complete robot and it looks like this.



![Image](https://github.com/WROMagnet/Magnet-Robots/assets/166937907/e4c3289d-1bd9-46e9-898e-5a9dad5bb31d)

- Now we will be assembling the robot with a single floor and changing the model and then making the turning mechanism. We have already finished assembling the single-story model.



![Image](https://github.com/WROMagnet/Magnet-Robots/assets/166937907/9bb389dd-375f-4fbc-a52a-89d121c08d72)

- Now we will assemble the turning mechanism and mount it on the robot.

**Comment by WROMagnet** [comment]
We had complications when looking for a model of the turning mechanism but we managed to make a model that complied with the measurements that worked and was effective. What we used to make the turn was a 180 degree servo motor. Here we will be demonstrating the final model of our robot.



![Image](https://github.com/WROMagnet/Magnet-Robots/assets/166937907/c222d202-f22f-4b57-99b5-b9f6130004a1)



https://github.com/WROMagnet/Magnet-Robots/assets/166937907/7ba2ac53-0e1e-4a58-9319-715aba41073a




_With this we finish this day tomorrow we will start configuring the color sensors and after configuring it we will mount them to the Prizm and tomorrow we will also be chopping a little of the robot because the sides of the chassis affect the rotation of the turning mechanism so we will chop a little of the chassis so that the turning mechanism can rotate further to the sides._
